docs/en/guides/pipeline-integrations/development/sgtk-developer-bootstrapping.md

```diff
@@ -0,0 +1,409 @@
+user = authenticator.create_script_user(
+ api_script="Script Name",
+ api_key="4e48f....<use the key from your Shotgun site>"
+)
+
+# Tells Toolkit which user to use for connecting to Shotgun.
+sgtk.set_authenticated_user(user)
+```
+
+## Part 4 - Bootstrapping an engine
+
+Now you have an authenticated sgtk API, you can start the bootstrapping process.
+You can find a lot of information on the bootstrap API in our [reference docs](https://developer.shotgunsoftware.com/tk-core/initializing.html#bootstrap-api).
+
+The bootstrapping process at a high level essentially performs the following steps:
+
+1. Retrieves or locates the Toolkit configuration folder.
+2. Ensures that the configuration dependencies such as the apps and engines are present in the bundle cache. 
+If they are not present, and they are using cloud-based descriptors such as `app_store`, or `shotgun` then it will download them to the bundle cache.
+3. Swaps out the current loaded sgtk core for the one appropriate to the config.
+4. Initializes the engine, apps, and frameworks.
+
+
+{% include info title="Note" content="Usually bootstrapping should take care of everything that is needed for that engine to run successfully.
+However, in some situations, the engine may have specific setup requirements that fall outside of the bootstrap process, and must be handled separately." %}
+
+
+### Bootstrap Preparation
+To bootstrap, you must first create a [`ToolkitManager`](https://developer.shotgunsoftware.com/tk-core/initializing.html#toolkitmanager) instance.
+
+```python
+mgr = sgtk.bootstrap.ToolkitManager()
+```
+
+For Toolkit to bootstrap, it needs to know at least the entity, plugin id, and engine.
+This guide won't cover all the available parameters and options, as that's covered in the [reference documentation](https://developer.shotgunsoftware.com/tk-core/initializing.html#bootstrap-api).
+
+#### Plugin ID
+
+You can define the plugin id by passing a string to the `ToolkitManager.plugin_id` parameter before calling the bootstrap method.
+In this guide, you will be bootstrapping the `tk-shell` engine so you should provide a suitable plugin id name following the conventions described in the reference docs. 
+```python
+mgr.plugin_id = "basic.shell"
+```
+
+#### Engine
+If your goal is to launch an app or run Toolkit code in a standalone python environment outside of software such as Maya or Nuke, 
+then `tk-shell` is the engine you will want to bootstrap into. 
+
+If you are wanting to run Toolkit apps within a supported Software then you will want to pick the appropriate engine, such as `tk-maya` or `tk-nuke`.
+This parameter is passed directly to the [`bootstrap_engine` method](https://developer.shotgunsoftware.com/tk-core/initializing.html#sgtk.bootstrap.ToolkitManager.bootstrap_engine).
+
+```python
+engine = mgr.bootstrap_engine("tk-shell", ...
+```
+
+#### Entity
+The entity can be of any entity type that the configuration is set up to work with. 
```

Comment:
I opted to add your suggested sentance and made tweaks, rather than replace mine:
What do you think?

> The [`ToolkitManager.bootstrap_engine()`](https://developer.shotgunsoftware.com/tk-core/initializing.html#sgtk.bootstrap.ToolkitManager.bootstrap_engine) methods `entity` parameter, is used to set the [context](https://developer.shotgunsoftware.com/tk-core/core.html#context) and therefore [environment](https://developer.shotgunsoftware.com/tk-core/core.html?highlight=environment#module-pick_environment) for the launched engine.
The entity can be of any entity type that the...